buildpack.toml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018-2021 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+api = "0.6"
+
+[buildpack]
+  description = "A Cloud Native Buildpack that provides/does Quarkus."
+  homepage = "https://example.com/"
+  id = "paketo-buildpacks/quarkus"
+  keywords = ["quarkus"]
+  name = "Paketo Quarkus Buildpack"
+  version = "{{.version}}"
+
+  [[buildpack.licenses]]
+    type = "Apache-2.0"
+    uri = "https://github.com/paketo-buildpacks/quarkus/blob/main/LICENSE"
+
+[metadata]
+  include-files = ["LICENSE", "NOTICE", "README.md", "bin/build", "bin/detect", "bin/main", "buildpack.toml"]
+  pre-package = "scripts/build.sh"
```

Comment:
Need to add this script. How it should look like?

Reply:
No worries, it's standard. I can add it.